pkgs/tools/networking/alpaca-proxy/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "alpaca-proxy";
+  version = "2.0.9";
+
+  src = fetchFromGitHub {
+    owner = "samuong";
+    repo = "alpaca";
+    rev = "v${version}";
+    hash = "sha256-Rf8//4FeruVZZ//uba80z20XGUxycwF91Aa09fosRXI=";
+  };
+
+  vendorHash = "sha256-JEiHgyPJvWmtPf8R4aX/qlevfZRdKajre324UsgRm5Y=";
+
+  ldflags = [
+    "-s"
+    "-w"
```

Comment:
Other go based packages need to inject the version information via ldflags, is the version information correctly represented for this package?

Reply:
Add `BuildVersion` override with ldflags. Thanks for pointing out :D